scripts/erlang-setup.ps1

```diff
@@ -12,24 +12,22 @@ $TEMPLATES_DIR = Join-Path $PSScriptRoot "templates"
 
 
 function Install-VCredist {
-    Write-Output "Install VCredist 2013"
-    $installerPath = Join-Path $env:TEMP "vcredist_2013_x64.exe"
-    Start-ExecuteWithRetry { Invoke-WebRequest -UseBasicParsing -Uri $VCREDIST_2013_URL -OutFile $installerPath }
+    Write-Log "Install VCredist 2013"
+    $installerPath = Join-Path $AGENT_BLOB_DEST_DIR $VCREDIST_2013_INSTALLER
+    Write-Log "Install VCredist 2013 from $installerPath"
     $p = Start-Process -Wait -PassThru -FilePath $installerPath -ArgumentList @("/install", "/passive")
     if ($p.ExitCode -ne 0) {
         Throw ("Failed install VCredist 2013. Exit code: {0}" -f $p.ExitCode)
     }
-    Write-Output "Finished to install VCredist 2013 x64"
+    Write-Log "Finished to install VCredist 2013 x64"
     Remove-File -Path $installerPath -Fatal $false
 }
 
 function Install-Erlang {
     New-Directory -RemoveExisting $ERLANG_DIR
-    $erlangZip = Join-Path $env:TEMP "erlang.zip"
-    Write-Output "Downloading the Windows Erlang runtime zip"
-    Start-ExecuteWithRetry { Invoke-WebRequest -UseBasicParsing -Uri $ERLANG_URL -OutFile $erlangZip }
-    Write-Output "Extracting the Windows Erlang zip to $ERLANG_DIR"
-    Expand-Archive -LiteralPath $erlangZip -DestinationPath $ERLANG_DIR
+    $erlangZip = Join-Path $AGENT_BLOB_DEST_DIR "erlang.zip"
```

Comment:
The `erlang.zip` is not included in the blob. A future PR to https://github.com/Microsoft/mesos-jenkins needs to address this.

The erlang from http://dcos-win.westus.cloudapp.azure.com/downloads/erl8.3.zip is needed to be included.

We simply need to add this to the dictionary of source files from [here](https://github.com/Microsoft/mesos-jenkins/blob/412ef8136c19cb1ccbc1d38ffd835d7564a859e2/AgentBlob/GenerateAgentBlob.ps1#L26).